notebooks/gridspecLayout.ipynb

```diff
@@ -0,0 +1,109 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Which multiplication table do you want to learn?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [
```

Comment:
I was quickly checking the other examples, and their outputs are removed from the notebook. So we should probably do the same here.

Reply:
Thanks @jtpio! I removed the cell outputs and added a notebook description. Will open a new PR for the docs.